samples/bluetooth/hci_spi/README.rst

```diff
@@ -0,0 +1,19 @@
+Bluetooth: HCI SPI
+##################
+
+Overview
+********
+
+Expose Zephyr Bluetooth Controller support over SPI to another device/CPU using
+the Zephyr SPI HCI transport protocol (similar to BlueNRG).
+
+
+Requirements
+************
+
+* A board with SPI and BLE support
+
+Building and Running
+********************
+
+See :ref:`bluetooth setup section <bluetooth_setup>` for details.
```

Comment:
@mbolivar : Sorry for the confusion, my comment was mostly for @dbkinder , showing that your short README for hci_spi is on par with ones hci_uart and hci_usb, and *they* have issues like that (which all need to be fixed, but hopefully in separate pull requests, and not blocking this patch ;-) ).